recipes/pyrobuf/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyrobuf" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7170d8060b33d6fea6a70cef937483267fbfc6a6b73a064700fc1b073de9eda6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - pyrobuf = pyrobuf.__main__:main
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - pip
+    - msinttypes  # [win]
+    - jinja2 >=2.8
+    - cython >=0.23
+    - pytest-runner
```

Comment:
Is ``pytest-runner`` really necessary to build the package? Could you please confirm that?

Reply:
Usually if it is a dependency yes. Though it appears to be verified below.

Reply:
I was surprised too, I actually tried to build it without it first, but it turned out to be important.